services/workflows-service/src/storage/storage.controller.external.ts

```diff
@@ -57,15 +59,37 @@ export class StorageControllerExternal {
   @common.Get('/:id')
   async getFileById(@Param('id') id: string, @Res() res: Response) {
     // currently ignoring user id due to no user info
-    const fileNameOnDisk = await this.service.getFileNameById({
+    const persistedFile = await this.service.getFileNameById({
       id,
       userId: '',
     });
+    if (!persistedFile) {
+      throw new errors.NotFoundException('file not found');
+    }
 
-    if (!fileNameOnDisk) {
+    return res.send(persistedFile);
+  }
+
+  // curl -v http://localhost:3000/api/storage/content/1679322938093
+  @common.Get('/content/:id')
+  async fetchFileContent(@Param('id') id: string, @Res() res: Response) {
```

Comment:
Please change `fetchFileContent` to `getFileContentById` for consistency. :pray:

Reply:
in this case, it fetches the file's content itself. and not getting the file object out of the databsase.
therefore I am not sure if consistency is the best practice here.
it might. but it does not consistent to other methods.
therefore should the naming be consistent then?